tools/mob_suite/mob_recon.xml

```diff
@@ -12,78 +13,133 @@
     #set $named_input = re.sub(r'(\s|\(|\)|:|!)', '_', str($input.element_identifier)+'.fasta')
     ln -s '$input' '$named_input' &&
     
-    mob_recon  --num_threads \${GALAXY_SLOTS:-4}  --infile '${named_input}'
+
+    mob_recon  --num_threads \${GALAXY_SLOTS:-4}  --infile '${named_input}' --run_typer
+
     #if str($adv_param.unicycler_contigs) == "True":
       --unicycler_contigs 
     #end if   
-    #if str($adv_param.run_circlator) == "True":
-      --run_circlator 
-    #end if 
-    #if str($adv_param.min_length_condition.min_length_param) == "True":
-      --min_length '${adv_param.min_length_condition.min_length_value}'
+
+    #if str($adv_param.run_overhang) == "True":
+      --run_overhang
     #end if 
-    --run_typer --min_rep_evalue '${adv_param.min_rep_evalue}'
+    
+    #if $adv_param.debug:
+      --debug
+    #end if
+
+    #if $adv_param.plasmid_db
+   --plasmid_db $adv_param.plasmid_db
+   #end if
+
+   #if $adv_param.plasmid_mash_db
+   --plasmid_mash_db '$adv_param.plasmid_mash_db'
+   #end if
+
+   #if $adv_param.plasmid_meta
+   --plasmid_meta '$adv_param.plasmid_meta'
+   #end if
+
+   #if $adv_param.repetitive_mask
+   --repetitive_mask '$adv_param.repetitive_mask'
+   #end if
+
+   #if $adv_param.plasmid_mob
+   --plasmid_mob $adv_param.plasmid_mob
+   #end if
+   
+   #if $adv_param.plasmid_mpf
+   --plasmid_mpf '$adv_param.plasmid_mpf'
+   #end if
+
+   #if $adv_param.plasmid_orit
+   --plasmid_orit '$adv_param.plasmid_orit'
+   #end if
+
+    --min_length '${adv_param.min_length}'  
+    --min_rep_evalue '${adv_param.min_rep_evalue}'
     --min_rep_evalue '${adv_param.min_rep_evalue}'
     --min_mob_evalue '${adv_param.min_mob_evalue}'
     --min_con_evalue '${adv_param.min_con_evalue}'
     --min_rep_ident '${adv_param.min_rep_ident}'
     --min_mob_ident '${adv_param.min_mob_ident}'
     --min_con_ident  '${adv_param.min_con_ident}'
     --min_rpp_ident   '${adv_param.min_rpp_ident}'
+
+    --min_rep_cov '${adv_param.min_rep_cov}'
+    --min_mob_cov '${adv_param.min_mob_cov}'
+    --min_con_cov '${adv_param.min_con_cov}'
+    --min_rpp_cov '${adv_param.min_rpp_cov}'
     --outdir 'outdir' &&
     mkdir ./outdir/plasmids && (mv outdir/plasmid*.fasta ./outdir/plasmids 2> /dev/null || true)
   ]]>  
   </command>
   <inputs>
     <param name="input" type="data" format="fasta" label="Input" help="FASTA file with contig(s)"/>
     <section name="adv_param" title="Advanced parameters" expanded="False">
-      <param name="unicycler_contigs" label="Check for circularity flag generated by unicycler in contigs fasta headers" type="select" value="True">
+      <param name="unicycler_contigs" label="Check for circularity flag generated by unicycler in contigs fasta headers?" type="select" value="True">
```

Comment:
Couldn't you make this a boolean value as you did with the `debug` param below?

Reply:
Yes, sure. I think toggle switch is easier. I just left it for a previous version. I will make it boolean.